src/deluge/model/global_effectable/global_effectable.cpp

```diff
@@ -295,36 +295,34 @@ bool GlobalEffectable::modEncoderButtonAction(uint8_t whichModEncoder, bool on,
 }
 
 int32_t GlobalEffectable::getKnobPosForNonExistentParam(int32_t whichModEncoder, ModelStackWithAutoParam* modelStack) {
-	int displayLevel = -64;
+	int current;
```

Comment:
Current should be initialized (for example to zero) since there are paths that don't set it